src/NServiceBus.Newtonsoft.Json/NServiceBus.Newtonsoft.Json.csproj

```diff
@@ -11,6 +11,6 @@
   <ItemGroup>
     <None Include="..\..\GitVersion.yml" Link="GitVersion.yml" />
     <PackageReference Include="Newtonsoft.Json" Version="[10.0.3, 11.0.0)" />
-    <PackageReference Include="NServiceBus" Version="[7.0.0-*, 8.0.0)" />
+    <PackageReference Include="NServiceBus" Version="[7.0.0-beta*, 8.0.0)" />
```

Comment:
this is redundant. a rebuild would have sufficed

Reply:
reverted.